package.json

```diff
@@ -23,6 +23,9 @@
       "import": "./lib/marked.esm.js",
       "default": "./lib/marked.cjs"
     },
+    "./lib/marked.cjs": "./lib/marked.cjs",
+    "./lib/marked.esm": "./lib/marked.esm.js",
+    "./lib/marked.umd": "./lib/marked.umd.js",
```

Comment:
**question (non-blocking):** I'm a little unsure on the naming here. Should the exports include file types?